pkg/networkutils/network.go

```diff
@@ -80,6 +80,11 @@ const (
 	maxRetryRouteAdd = 5
 
 	retryRouteAddInterval = 5 * time.Second
+
+	// number of attempts to find an ENI by MAC address after it is attached
+	maxAttemptsLinkByMac = 5
+
+	retryLinkByMacInterval = 5 * time.Second
```

Comment:
I would like to refactor this to use exponential backoff instead, but this change is consistent with the current code and looks good for now. I'll create an issue for that.